### ISSUE TITLE
Some css styling will break for all previous builds after pushing(with now.sh)

### ISSUE THREAD

**Issue by yassinebridi** [zeit/next.js#8239]
# Bug report

## Describe the bug

Some css styles stops working for **all builds**(now.sh), this behavior only occurs when i push a new production build, and weirdly only after midnite(never had this problem when pushing during the day)

## Expected behavior

For the styles to work properly at all times.

## Screenshots

This build https://frontend-2xb1cn2wa.now.sh/ , looks like this at the time of writing this(00:56 AM UTC+1):
https://storage.googleapis.com/openscreenshot/S/X/H/Hk0UayHXS.png
but after some hours, without pushing any code. it will look normal like this:
https://storage.googleapis.com/openscreenshot/H/7/r/SJ5h0Jr7H.png
(after an hour or so i would expect it to behave normally, as it happened quite sometimes)

## System information
- Browser: chrome, 76.0.3809.87
- Version of Next.js: 9.0.2

## Additional context

**Important notes**:
1- When this behavior occurs, all the previous builds will have the same issue.
2- I ran this from a private window, to make sure there would be no caching problem.

config files:
-package.json: https://gist.github.com/yassinebridi/a9e2b15a033b4ccd6464622915b70b94
-next.config: https://gist.github.com/yassinebridi/94325a3a919acbde8161611b1b261c78

*I honestly don't know if this is debuggable, but i had to let you know*

**Comment by developit** [comment]
Can you also post your `.babelrc` / `babel.config.js`?

**Comment by yassinebridi** [comment]
i have only one preset in .babelrc
```babelrc
{
  "presets": ["next/babel"]
}
```

**Comment by Timer** [comment]
Closing as fixed by #8626

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
